src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java

```diff
@@ -194,6 +194,38 @@ public void SignUp_007_signUpWithFacebook() {
     base.logOut(wikiURL);
   }
 
+  @Test(groups = {"SignUp_008", "SignUp"})
+  public void SignUp_008_signupJapaneseUser() {
+    String wikiURLJapanese = "http://ja.ja-test.wikia.com/";
+    WikiBasePageObject base = new WikiBasePageObject(driver);
```

Comment:
If SignUpPageObject extends WikiBasePageObject you can use this:
```
SignUpPageObject signUp = new SignUpPageObject();
signUp.navigateToSpecialSignUpPage(wikiURLJapanese);
```
It will give you chance to remove this line:
```
WikiBasePageObject base = new WikiBasePageObject(driver);
```